packages/amplify-schema-validator/src/index.ts

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable global-require */
+
+import { parse } from 'graphql';
+
+const allValidators = [
+  require('./validators/index-scalar-types').validate,
+  require('./validators/relation-required-with-belongs-to').validate,
+  require('./validators/two-many-to-many-locations').validate,
```

Comment:
why use requires here instead of `import { blahValidator } from './validators/index-scalar-types';`

Reply:
Initially I was thinking of writing this file at build time to guarantee that all validators are run and limit the number of files to be touched by contributors to just their validator and their test. But since that is impractical, I reluctantly determined it's ok to also have contributors touch this file in a one-liner-y kind of way. The import strategy requires touching the file in two places.

But it's not a big deal, I don't think there will be too many merge conflicts in this file, and if there are, they would be very easily resolved.